src/qe_tools/parsers/_input_base.py

```diff
@@ -207,6 +207,12 @@ def __init__(self, content, *, qe_version=None, validate_species_names=True):
             qe_version=self._qe_version
         )
 
+    def as_dict(self) -> dict:
+        """docstring"""
```

Comment:
Beautiful docstring ;) Would be good to add a real one. Same goes for `_pw_input`.

Reply:
Oops 😅 Will add.